Dockerfile

```diff
@@ -85,6 +97,7 @@ EOT
 
 FROM ${distroless_image} AS base
 COPY --from=builder /usr/share/misc/magic.mgc /usr/share/misc/magic.mgc
+COPY --from=builder /copy/bi[n]/ /bin/
```

Comment:
Is this a character class? Wouldn't it here be sufficient to simply have a /copy/bin/ ?

Reply:
it was a trick to avoid a copy error in case the source doesn't exist. but it is removed anyway since not needed anymore